shared/chat/conversation/attachment-popup/index.native.js

```diff
@@ -46,18 +45,9 @@ const AttachmentPopup = ({message, detailsPopupShowing, isZoomed, onCloseDetails
           <Text type='BodySmall' style={{color: globalColors.black, marginTop: globalMargins.large}}>Your device can not preview this file.</Text>
         </Box>
         <Box style={styleHeaderFooter}>
-          <Icon type='iconfont-ellipsis' onClick={detailsPopupShowing ? onCloseDetailsPopup : onOpenDetailsPopup} />
```

Comment:
Can you now delete the `onCloseDetailsPopup`/`onOpenDetailsPopup` stuff in `attachment-popup/container.js`?